pandas/io/tests/test_parse_sql.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
I thought about that, but they don't seem to overlap in fact in anyway.  The doc-string that I added helps to make that clear I believe.